doc/getting_started/modern-gl.rst

```diff
@@ -273,18 +273,18 @@ and not the fixed one.
 An intro to modern OpenGL
 -------------------------
 
-OpenGL has been around a long time, and from reading all the accumulated layers
+OpenGL has been around a long time and from reading all the accumulated layers
```

Comment:
If we're being picky/pedantic, is this grammatically correct? The "and" after the comma throws me off as to whether the comma is appropriate.

Reply:
The comma seemed not threw me a bit off while I was reading the text. I don't see a reason to put a comma there since the word 'and' already does its work and connects the two sentences. 

It cannot be a serial comma either, because there is nothing being enumerated.

Reply:
Ha, sorry, I thought this was the opposite change and you were adding the comma. I agree with this. Thanks.